account_cutoff_start_end_dates/__init__.py

```diff
@@ -1 +1,53 @@
 from . import models
+
+from openupgradelib import openupgrade
+from psycopg2 import sql
+
+
+def module_migration(cr):
+    account_cutoff_line_table = "account_cutoff_line"
+    account_cutoff_prepaid = "account_cutoff_prepaid"
+    account_cutoff_accrual_dates = "account_cutoff_accrual_dates"
+    account_cutoff_start_end_dates = "account_cutoff_start_end_dates"
+    if openupgrade.is_module_installed(cr, account_cutoff_accrual_dates):
+        openupgrade.update_module_names(
+            cr,
+            [
+                (account_cutoff_accrual_dates, account_cutoff_start_end_dates),
+            ],
+            merge_modules=True,
+        )
+    if openupgrade.is_module_installed(cr, account_cutoff_prepaid):
+        openupgrade.update_module_names(
+            cr,
+            [
+                (account_cutoff_prepaid, account_cutoff_start_end_dates),
+            ],
+            merge_modules=True,
+        )
+        openupgrade.logged_query(
```

Comment:
The old column should also be removed, and also filters and translations have to be updated; I have found out in https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0:
> Remove all the `oldname` field attributes in the code. If they were added in previous version, they have served their function any way, and now in this version it's not supported, so if you have the need, create a migration script and use openupgradelib's `rename_fields` method.

So it should be enough to call `rename_fields` (https://github.com/OCA/openupgradelib/blob/c35c0e4c5cdd445b23d496e569a52921ac8c4a85/openupgradelib/openupgrade.py#L575); can you execute `rename_fields` instead of just copying the values with the below queries?

Reply:
done

Reply:
`rename_fields` renames the column `prepaid_days` to `cutoff_days` so `cutoff_days` should not exist when you call `rename_fields`, otherwise the following is raised:
> ERROR:  column "cutoff_days" of relation "account_cutoff_line" already exists

Can you remove the column creation?

Reply:
Thanks, I see the new edits, but they should be in the [MIG] commit.
Moreover, why is this PR adding more than 200 commits? It should only add one commit with your changes to https://github.com/akretion/account-closing/tree/14-account_cutoff_start_end_dates